### ISSUE TITLE
Toast documentation missing props

### ISSUE THREAD

**Issue by davicostalf** [vtex/shoreline#1812]
### Problem

There are no props listed in the Toast documentation. It’s worth investigating what was the issue in the generator and if other components have also been affected.

### Expected behavior

Every components should have their props in the documentation

### Package Manager

pnpm

**Comment by kevinch** [comment]
Yes! I just found out and I think I got the solution. But the build of the docs is failing here. I created https://github.com/vtex/shoreline/issues/1853 today for this.

If we can fix it, then It might enter the https://github.com/vtex/shoreline/pull/1795 PR for the docs.

**Comment by lafray** [comment]
@kevinch I'll change the status to active so that another unsuspecting person doesn't end up starting this issue, since you have the fix done

**Comment by kevinch** [comment]
Fix within https://github.com/vtex/shoreline/pull/1875

**Comment by davicostalf** [comment]
Props are already in the documentation, but we still need to document the toast function.
